code/__HELPERS/unsorted.dm

```diff
@@ -653,7 +653,16 @@ Turf and target are seperate in case you want to teleport some distance from a t
 	return .
 
 /proc/key_name_admin(var/whom, var/include_name = 1)
-	return key_name(whom, 1, include_name)
+	var/message = "[key_name(whom, 1, include_name)](<A HREF='?_src_=holder;adminmoreinfo=\ref[whom]'>?</A>)[isAntag(whom) ? " (ANTAG)" : ""] ([admin_jump_link(whom, "holder")])"
+	return message
+	
+/proc/get_mob_by_ckey(key)
+	if(!key)
+		return
+	var/list/mobs = sortmobs()

```

Comment:
I'm pretty sure that sorting `mob_list` here, just before searching it for a single mob by its key, does nothing but slow this proc down.